pkg/test/resourcefixture/testdata/basic/bigtable/v1beta1/bigtableinstance/bigtableinstanceautoscaling2/_http.log

```diff
@@ -103,66 +103,18 @@ OK
     "originalRequest": {
       "clusters": {
         "us-central1-a": {
-          "clusterConfig": {
```

Comment:
I like what we're adding but a little concerned that this means we are losing coverage on our tests for a few field types.